README.md

```diff
@@ -50,7 +50,11 @@ Version | Supported Laravel Versions | Support
 1.1.x | 5.2.x |  EOL
 1.2.x | 5.2.x, 5.3.x | Security releases
 1.3.x | 5.4.x | Bugfix and security releases
-1.4.x | 5.5.x | New features
+\>=1.4.0 | 5.5.x | New features
+\>=1.4.3 | 5.5.x, 5.6.x | New features
+\>=1.4.8 | 5.5.x, 5.6.x, 5.7.x | New features
+\>=1.4.10 | 5.5.x, 5.6.x, 5.7.x, 5.8.x | New features
```

Comment:
I don't think we need to repeat ourself in this column. Change to something like this: ?

Version | Supported Laravel Versions | Support
:---------|:----------|:----------
1.0.x |  5.1.x | EOL
1.1.x | 5.2.x |  EOL
1.2.x | 5.2.x, 5.3.x | Security releases
1.3.x | 5.4.x | Bugfix and security releases
\>=1.4.0 | 5.5.x | New features
\>=1.4.3 | 5.6.x | New features
\>=1.4.8 | 5.7.x | New features
\>=1.4.10 | 5.8.x | New features

Reply:
Actually, we should use `~` instead of `>=`, since 5.5 is not supported in 1.5.

https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

Reply:
Will do, I'll get this later today.

Reply:
I have added a similar table here: https://github.com/laravel-doctrine/acl

Reply:
Does 1.5 not support 5.6, 5.7 and 5.8 also, or just 5.5?

Reply:
1.5 is for ^6.0.